Sources/Utilities/KMZendeskChatHandler.swift

```diff
@@ -233,5 +319,56 @@ public class KMZendeskChatHandler {
         }
         return true
     }
+    
+    func updateLastMessageCreatedTime() {
+        let channelKey = NSNumber(value: Int(groupId) ?? 0)
+        guard channelKey != 0 else {
+            print("Failed to fetch messages for group Id")
+            return
+        }
+        
+        fetchMessages(channelKey: channelKey){ messages,error,userArray in
+            guard let messageList = messages,
+                  let almessages = messageList.reversed() as? [ALMessage] else {
+                return
+            }
+            self.lastUserMessageCreatedTime = almessages.last?.createdAtTime ?? 0
+        }
+    }
+    
+    func resetLocal() {
```

Comment:
When a new thread is created, then also we need to call this, no?
Also, change the name of this var.